resources/tests/static_cpu_templates/t2cl.json

```diff
@@ -90,7 +90,7 @@
   "msr_modifiers": [
     {
       "addr": "0x10a",
-      "bitmap": "0b0000000000000000000000000000000000000000000000000000000011101011"
+      "bitmap": "0b00000000000000000000000000000000000000000000x0000000000011101x11"
```

Comment:
this passes the bit 2 through as well, but the commit message does not mention it.

Reply:
good catch! we should mention it in the commit message! will fix.

Reply:
fixed.